test/e2e/node/pre_stop.go

```diff
@@ -78,6 +79,8 @@ func testPreStop(c clientset.Interface, ns string) {
 	podOut, err := c.CoreV1().Pods(ns).Get(podDescr.Name, metav1.GetOptions{})
 	framework.ExpectNoError(err, "getting pod info")
 
+	podUrl := net.JoinHostPort(podOut.Status.PodIP, "8080")
```

Comment:
golint is failing, this should be:
```suggestion
	podURL := net.JoinHostPort(podOut.Status.PodIP, "8080")
```

Reply:
amended your suggestion to the current commit